dockerfiles/alpine/is/Dockerfile

```diff
@@ -106,14 +108,9 @@ RUN \
     && apk add unzip \
     && apk add wget
 
-## set the user and work directory
-USER ${USER_ID}
-WORKDIR ${USER_HOME}
-
-COPY --chown=wso2carbon:wso2 ${WSO2_SERVER}.zip ${USER_HOME}/
-
 RUN \
-    unzip -d ${USER_HOME} ${WSO2_SERVER}.zip \
+    wget -O ${WSO2_SERVER}.zip "${WSO2_SERVER_DIST_URL}" \
```

Comment:
@SujanSanjula96 did we decide to remove wget from master? And to allow only copying?